intellij/src/main/kotlin/motif/intellij/provider/ScopeNavigationLineMarkerProvider.kt

```diff
@@ -72,26 +74,31 @@ class ScopeNavigationLineMarkerProvider : LineMarkerProvider, MotifProjectCompon
 
     override fun getLineMarkerInfo(element: PsiElement): LineMarkerInfo<PsiElement>? {
         val graph: ResolvedGraph = graph ?: return null
-        if (element is PsiClass && isMotifScopeClass(element)) {
-            val scopeEdges: Array<ScopeEdge>? = getParentScopes(element.project, graph, element)
+        val psiClassElement = element.toPsiClass()
```

Comment:
Test updates?

Reply:
No relevant existing tests, line markers are not usually tested it seems.